### DISCUSSION TITLE
Error whilst rebuilding

### DISCUSSION THREAD

**Discussion by Oriantar** [pretix/pretix#3766] [Q&A] [answered]
Hi iam getting a Internal Server Error and dont really know how to fix it.

When iam issueing the command: python -m pretix rebuild

This is what i get:
```
CommandError: An error occurred during rendering pretixcontrol/checkin/list_edit.html:
/var/pretix/venv/lib/python3.7/site-packages/pretix/static.dist/pretixcontrol/js/ui/checkinrules/datetimefield.255b93e5448f.vue → tmp/tmp1zqd45dh.js...
[!] (plugin babel) TypeError: unknown file: isTSSatisfiesExpression is not a function
```

**Comment by raphaelm** [comment]
Never seen that before so far. What is your nodejs version?

**Comment by raphaelm** [comment, marked as answer]
I also see you using Python 3.7. pretix requires at least 3.9. Not related to this issue probably, but either you also run a really old version of pretix or it's not going to work either way.

**Reply by Oriantar** [reply]
After updating the Venv to 3.12 and the postgres it worked thanks!
